src/test/java/com/wikia/webdriver/testcases/desktop/categoriestests/DynamicCategoryPageTests.java

```diff
@@ -0,0 +1,128 @@
+package com.wikia.webdriver.testcases.desktop.categoriestests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.communities.desktop.components.categories.CategoryLayoutSelector;
+import com.wikia.webdriver.elements.communities.desktop.components.categories.PaginationControls;
+import com.wikia.webdriver.elements.communities.desktop.pages.categories.DynamicCategoryPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
+
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+@Test(groups = {"Category, Oasis_Category"})
+@Execute(onWikia = "category")
+public class DynamicCategoryPageTests extends NewTestTemplate {
+
+
+  public void dynamicCategoryAnonNotAbleToChangeLayout() {
+    new ArticlePageObject().open("Category:200elements");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Category:200elements" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)